static-site/src/components/Footer/sitemap.jsx

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import styled from "styled-components";
+import Link from 'next/link'
+import { FaExternalLinkAlt } from "react-icons/fa";
+import * as Styles from "../splash/styles";
+import { BigSpacer } from "../../layouts/generalComponents";
+
+
+const UnstyledList = styled.ul`
+  list-style: none;
+`
+
+const ListItem = styled(Styles.FocusParagraph)`
+  line-height: 0.7;
+`
+
+const UncoloredLink = styled.span`
+  & a {
+    color: #000 !important;
+  }
+`
+
+const FooterIcon = styled.span`
+  margin-left: 2px;
+  font-size: 12px;
+`
+
+export const SiteMap = () => {
+  return  (
+    <>
+      <div className="col-md-3">
+        <BigSpacer />
+        <Styles.H3 $left>Resources</Styles.H3>
+        <UnstyledList>
+          <li>
```

Comment:
To reduce duplication like this we can define the links in an array and make use of [map within JSX](https://legacy.reactjs.org/docs/lists-and-keys.html#embedding-map-in-jsx), e.g.:

```js
// earlier in the file
const RESOURCES = [
  {name: 'Core pathogens', href: '/pathogens'},
  {name: 'SARS-CoV-2', href: '/sars-cov-2'},
  ...
];
```

and then swap `<UnstyledList>...</UnstyledList>` with
```js
<UnstyledList>
  {RESOURCES.map((resource) => (
    <li>
      <ListItem>
        <UncoloredLink>
          {resource.href.startswith('http') ? (
            <Link href={resource.href}  rel="noopener noreferrer" target="_blank">
              {resource.name} <FooterIcon><FaExternalLinkAlt/></FooterIcon>
            </Link>
          ) : (
            <Link href={resource.href}>
              {resource.name}
            </Link>
          )}
        </UncoloredLink>
      </ListItem>
    </li>
  ))}
</UnstyledList>
```

You can go further if you want and define everything together, e.g.:
```js
const LINKS = [
  {title: 'Resources', links: [{name: ...}]},
  {title: 'Tools', links: [{name: ...}]},
  ...
]
```
and then use nested `map`s to further simplify things.

Reply:
Nice!